mne/io/tests/test_reference.py

```diff
@@ -261,6 +261,23 @@ def test_add_reference():
     raw = Raw(fif_fname, preload=True)
     picks_eeg = pick_types(raw.info, meg=False, eeg=True)
 
+    # add reference channel to Raw when no digitization points exist
+    raw = Raw(fif_fname, preload=True)
```

Comment:
to save a little bit of time you can do `raw = Raw(fif_fname).crop(0, 1).load_data()` here instead